code/modules/uplink/uplink_items/uplink_roles.dm

```diff
@@ -127,7 +127,7 @@
 /datum/uplink_item/role_restricted/goldenbox
 	name = "Gold Toolbox"
 	desc = "A gold planted plastitanium toolbox loaded with tools. Comes with a set of AI detection multi-tool and a pare of combat gloves."
-	item = /obj/item/storage/toolbox/gold_real
+	item = /obj/item/storage/toolbox/plastitanium/gold_real
 	cost = 3 // Has syndie tools + gloves + a robust weapon
```

Comment:
Actually, considering this is now a subtype of a toolbox that wrecks objects, could you buff the cost a little?
```suggestion
	cost = 5 // Has syndie tools + gloves + a robust weapon
```